contracts/root/withdrawManager/WithdrawManager.sol

```diff
@@ -384,6 +384,7 @@ contract WithdrawManager is WithdrawManagerStorage, IWithdrawManager {
   }
 
   function getExitableAt(uint256 createdAt) internal view returns (uint256) {
-    return Math.max(createdAt + 2 * HALF_EXIT_PERIOD, now + HALF_EXIT_PERIOD);
```

Comment:
just change the constant Var `HALF_EXIT_PERIOD`.

Reply:
Then it can't be used as a replacement for a currently deployed contract. We can't change storage.